subprojects/language-java/src/integTest/groovy/org/gradle/javadoc/JavadocIntegrationTest.groovy

```diff
@@ -178,16 +178,6 @@ Joe!""")
         nonSkippedTasks == [":javadoc"]
     }
 
-    def "ensure javadoc task does not change its inputs"() {
-        executer.withArgument("-Dorg.gradle.tasks.verifyinputs=true")
-        buildFile << """
-            apply plugin: 'java'
-        """
-        writeSourceFile()
-        expect:
-        succeeds("javadoc")
-    }
-
```

Comment:
Why remove this now?

Reply:
We don't have `org.gradle.tasks.verifyinputs` anymore, so the test is useless.